source/shared/cpp/ObjectModel/ActionSet.h

```diff
@@ -23,9 +23,8 @@ namespace AdaptiveCards
         const std::vector<std::shared_ptr<BaseActionElement>>& GetActions() const;
 
     private:
-        void PopulateKnownPropertiesSet() override;
+        void PopulateKnownPropertiesSet();
         std::vector<std::shared_ptr<AdaptiveCards::BaseActionElement>> m_actions;
-        ActionsOrientation m_orientation;
```

Comment:
>m_orientation [](start = 27, length = 13)

Are we calling this a fix to #3461? Or is that bug now tracking the possibility of adding orientation to the action set element?

Reply:
I'll mark 3461 as related :)

---
In reply to: [335689678](https://github.com/microsoft/AdaptiveCards/pull/3520#discussion_r335689678) [](ancestors = 335689678)